mmf/models/movie_mcan.py

```diff
@@ -24,7 +24,9 @@ def __init__(self, config):
         super().__init__(config)
         self.config = config
         self._global_config = registry.get("config")
-        self._datasets = self._global_config.datasets.split(",")
+        self._datasets = self._global_config.datasets
+        if isinstance(self._datasets, str):
```

Comment:
what is the alternative, should we assert/warning there?

Reply:
An alternative is not really needed, this usually happens when the config is loaded from a file or checkpoint. So, this is just handling that edge case.